shared/src/commonMain/graphql/extra.graphqls

```diff
@@ -0,0 +1,6 @@
+extend type Query
+@typePolicy(embeddedFields: "sessions")
+@fieldPolicy(forField: "sessions", paginationArgs: "first, after")
```

Comment:
This is the current way to make pagination work - ideally we'll find a way to make this less involved/verbose.